tests/components/spotify/test_sensor.py

```diff
@@ -0,0 +1,65 @@
+"""Tests for the Spotify sensor platform."""
+
+from unittest.mock import MagicMock, patch
+
+import pytest
+from spotifyaio import PlaybackState
+from syrupy import SnapshotAssertion
+
+from homeassistant.components.spotify import DOMAIN
+from homeassistant.const import STATE_UNKNOWN, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, load_fixture, snapshot_platform
+
+
+@pytest.mark.usefixtures("setup_credentials")
+async def test_entities(
+    hass: HomeAssistant,
+    mock_spotify: MagicMock,
+    mock_config_entry: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+) -> None:
+    """Test the Spotify entities."""
+    with patch("homeassistant.components.spotify.PLATFORMS", [Platform.SENSOR]):
+        await setup_integration(hass, mock_config_entry)
```

Comment:
We should really figure out something for this weird patch behavior we put on everywhere. IMHO integration setup should just be a fixture and this patching should not be needed.

Reply:
But I also don't find it ideal to have 5-10 extra mocks in the conftest, while 6-7 are only used in one test only

Reply:
> But I also don't find it ideal to have 5-10 extra mocks in the conftest, while 6-7 are only used in one test only.

Not sure how that is related. We can just set up the whole integration. As a matter of fact. Setting up the integration partly as done in these constructs may hide side-effects.

Reply:
I don't think I understand what you mean

Reply:
it is also not for now, I'm just not happy with seeing this pattern everywhere.